jaeger-apachehttpclient/src/main/java/com/uber/jaeger/httpclient/TracingRequestInterceptor.java

```diff
@@ -50,11 +49,11 @@ public void process(HttpRequest httpRequest, HttpContext httpContext)
     try {
       spanCreationInterceptor.process(httpRequest, httpContext);
 
-      Scope currentScope = tracer.scopeManager().active();
-      if (currentScope != null) {
-        onSpanStarted(currentScope.span(), httpRequest, httpContext);
+      Span clientSpan = (Span) httpContext.getAttribute(Constants.CURRENT_SPAN_CONTEXT_KEY);
+      if (clientSpan != null) {
+        onSpanStarted(clientSpan, httpRequest, httpContext);
       } else {
-        log.warn("Current scope is null; possibly failed to start client tracing span.");
+        log.warn("Current client span is null; SpanCreationRequestInterceptor possibly failed to start client tracing span.");
```

Comment:
I assume we already have a test for TracingRequestInterceptor, is it possible to add a check that log.warn is not called? With the expectation that without this change such check will fail.

Reply:
that sounds like a good idea

Reply:
@yurishkuro checking for logger state was definitely (not) a fun activity haha
ended up using a test-focused in-memory logger.